tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -20,6 +20,14 @@ Verify Resource Link Http status code
         Log To Console    ${idx}. ${href} gets status code ${status}
     END
 
+Go to RHODS Dashboard, Resources And Filter By enabled state
+  [Documentation]  check if it is possible to filter items by enable state or application
+  [Tags]  ODS-489
+  Click Link    Resources
+  sleep  10s
```

Comment:
would it be possible to replace the sleep with some other logic? maybe waiting for some page component to appear